doc/io.rst

```diff
@@ -921,6 +924,22 @@ pass to xarray::
     # read it back
     ds_gcs = xr.open_zarr(gcsmap)
 
+New in v0.16.2: general `fsspec`_ URLs are now parsed and the store set up for you
```

Comment:
What do you think about swapping the order of these — having a "from v0.16.2" first and then "for older versions..." after (if at all)

Reply:
done